src/routing_filter.c

```diff
@@ -595,7 +595,15 @@ routing_filter_add(cache          *cc,
 
          // Set the index_no
          // ALEX: for now the indices must fit in a single extent
-         debug_assert(index_no / addrs_per_page < pages_per_extent);
+         platform_assert((index_no / addrs_per_page < pages_per_extent),
+                         "index_no=%u, addrs_per_page=%lu"
+                         ", (index_no / addrs_per_page)=%lu"
+                         ", pages_per_extent=%lu",
+                         index_no,
+                         addrs_per_page,
+                         (index_no / addrs_per_page),
+                         pages_per_extent);
```

Comment:
Changed to `platform_assert()` and elaborated diag info ... as I was running into some bugs in this area before fingerprint / fp-array handling was eventually stabilized.